chartpress.py

```diff
@@ -1008,10 +1059,13 @@ def main(args=None):
     #   - push chart (--publish-chart, --extra-message)
     for chart in config["charts"]:
         forced_version = None
+        # TODO: maybe warn and switch default in the future?
+        use_chart_version = chart.get("useChartVersion", False)
+
         if args.tag:
             forced_version = args.tag
-        if args.reset:
-            forced_version = chart.get("resetVersion", "0.0.1-set.by.chartpress")
+        elif args.reset and not use_chart_version:
```

Comment:
Should this be `elif args.reset and use_chart_version` ?

Reply:
No, this branch is "resetting chart version from chartpress.yaml, ignoring what's in Chart.yaml"